From 871835ecc4ee380fe42d1ef2a58dca9dc4e9af5f Mon Sep 17 00:00:00 2001
From: Andrew Bauer <zonexpertconsulting@outlook.com>
Date: Wed, 21 Feb 2018 12:07:12 -0600
Subject: [PATCH] allow one to load custom environment variables

---
 packpack | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/packpack b/packpack
index 6f4c80f..abab604 100755
--- a/packpack/packpack
+++ b/packpack/packpack
@@ -128,6 +128,11 @@ chmod a+x ${BUILDDIR}/userwrapper.sh
 env | grep -E "^PRODUCT=|^VERSION=|^RELEASE=|^ABBREV=|^TARBALL_|^CHANGELOG_|^CCACHE_|^PACKAGECLOUD_|^SMPFLAGS=|^OS=|^DIST=" \
     > ${BUILDDIR}/env
 
+# Load user environment variables
+if [ -f "${USER_ENV_FILE}" ]; then
+    cat ${USER_ENV_FILE} >> ${BUILDDIR}/env
+fi
+
 #
 # Setup cache directory
 #
