--- a/distros/ubuntu1204/rules	2017-03-05 11:46:18.707725975 -0600
+++ b/distros/ubuntu1204/rules	2017-03-05 12:33:30.639678284 -0600
@@ -58,8 +58,10 @@
 
 override_dh_fixperms:
 	dh_fixperms
-	## 637685
-	chmod -c o-r $(CURDIR)/debian/zoneminder/etc/zm/zm.conf
+	#
+	# As requested by the Debian Webapps Policy Manual §3.2.1
+	chown root:www-data $(CURDIR)/debian/zoneminder/etc/zm/zm.conf
+	chmod 640 $(CURDIR)/debian/zoneminder/etc/zm/zm.conf
 
 override_dh_installinit:
 	dh_installinit --no-start
--- a/distros/ubuntu1604/rules	2017-03-05 11:46:18.707725975 -0600
+++ b/distros/ubuntu1604/rules	2017-03-05 12:35:43.960426310 -0600
@@ -58,8 +58,10 @@
 
 override_dh_fixperms:
 	dh_fixperms
-	## 637685
-	chmod -c o-r $(CURDIR)/debian/zoneminder/etc/zm/zm.conf
+	#
+	# As requested by the Debian Webapps Policy Manual §3.2.1
+	chown root:www-data $(CURDIR)/debian/zoneminder/etc/zm/zm.conf
+	chmod 640 $(CURDIR)/debian/zoneminder/etc/zm/zm.conf
 
 override_dh_installinit:
 	dh_installinit --no-start
